### PR TITLE
Rename :EnClasspath to :EnInstall

### PR DESCRIPTION
**Pull request by ches**
As per #303, this should hopefully be a lot less puzzling for a user’s first encounter with ensime-vim—the server classpath bootstrapping process is an implementation detail they shouldn’t have to understand beyond "install the ENSIME server, please".

### CODE REVIEW

**Comment by ches** [comment]
Oh, give me a heads up when ready to merge and I'll prepare a PR for the minor update that would be needed for https://ensime.github.io/ docs.

**Comment by ChrisCoffey** [comment]
LGTM 👍

File: ensime_shared/ensime.py [hunk 1]
```diff
@@ -648,8 +650,15 @@ def type_check_cmd(self, args, range=None):
...
-        self.log("en_classpath: in")
+    def en_install(self, args, range=None):
+        """Bootstrap ENSIME server installation.
+
+        Enabling the bootstrapping actually happens in the execute_with_client
+        decorator when this is called, so this function is a no-op endpoint for
+        the Vim command.
+        TODO: this is confusing...
+        """
+        self.log("en_install: in")
```
Comment (ChrisCoffey) [inline ensime_shared/ensime.py]: Can't wait for #297 to get rid of all these

**Comment by jvican** [comment]
LGTM, this is ready to merge right away. Let me know when you raise that PR and I'll click the button :wink:.

**Pull request merged**